pallets/utxo/src/lib.rs

```diff
@@ -640,6 +787,9 @@ pub mod pallet {
                         crate::script::verify(&tx, &input_utxos, index as u64, witness, lock)
                             .map_err(|_| "script verification failed")?;
                     }
+                    _ => {
+                        log::info!("TODO validate STAKE spending");
```

Comment:
Quite frankly, I was looking forward to seeing how this is implemented to understand how all this would work... too bad.